### ISSUE TITLE
Fix dialyxir warnings

### ISSUE THREAD

**Issue by mrinalwadhwa** [ockam-network/ockam#917]
Looks like there's a new release coming for Dialyxir, hopefully that fixes these.
https://github.com/jeremyjh/dialyxir/issues/423

```
warning: Mix.Project.compile/1 is deprecated. Use Mix.Task.run("compile", args) instead
  lib/dialyxir/project.ex:47: Dialyxir.Project.cons_apps/0

warning: Mix.Project.compile/1 is deprecated. Use Mix.Task.run("compile", args) instead
  lib/mix/tasks/dialyzer.ex:164: Mix.Tasks.Dialyzer.run/1
```

**Comment by jeremyjh** [comment]
Yes, these are fixed in 1.1.0, just released.

**Comment by mrinalwadhwa** [comment]
awesome, thank you 🙏
